LICENSE

```diff
@@ -1,6 +1,6 @@
 Apache Software License 2.0
 
-Copyright (c) 2021, Network to Code, LLC
+Copyright (c) 2025, Network to Code, LLC
```

Comment:
```suggestion
Copyright (c) 2021-2025, Network to Code, LLC
```

Reply:
The cookies just get updated to the "current" year and its updated that way across apps etc. Let me see if there is a way to make it work out. I'll update this PR though to get this one merged in.